packages/react-components/src/Status/index.tsx

```diff
@@ -10,15 +10,23 @@ import styled from 'styled-components';
 import { GenericCall } from '@polkadot/types';
 
 import AddressMini from '../AddressMini';
+import Button from '../Button';
 import Icon from '../Icon';
-import { classes } from '../util';
 import translate from '../translate';
+import { classes } from '../util';
 
 interface Props extends I18nProps {
   stqueue?: QueueStatus[];
   txqueue?: QueueTx[];
 }
 
+const STATUS_COMPLETE: QueueTxStatus[] = [
```

Comment:
One last request - rather just move this into a `constants.ts` file and import it in both `index.tsx` and `Queue.tsx`. (That we we have no duplication)

Reply:
Sorry. Shouldn't have bugged you for adding `constants.ts`. It was pretty obvious 😅